src/views/BoardEvents.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <v-container>
+    <app-bar
+      :back-to="`/board/${$route.params.boardId}`"
+    />
+    <event-list
+      :events="currentEvents"
+      all-events
+    />
+    <event-create-dialog
+      ref="eventCreateDialog"
+      :subjects="subjects"
+      :initial-date="new Date().toISOString().split('T')[0]"
+      :value="$route.name === 'BoardCreateEventAll' || $route.name === 'BoardEditEventAll'"
+      :edit="$route.name === 'BoardEditEventAll' || lastDialogState.edit"
+      :loading="!eventsAndSubjectsLoaded || !boardInfoLoaded"
+      :event="dialogEvent || lastDialogState.event"
+      :user-is-member="userIsMember"
+      @close="closeCreatorDialog()"
+    />
+    <v-dialog
+      :value="$route.name === 'BoardEventAll' && canViewBoard === true"
+      scrollable
+      max-width="500px"
+      persistent
+      no-click-animation
+      @click:outside="closeEventDetailsDialog()"
+      @keydown.esc="closeEventDetailsDialog()"
+    >
+      <event-details-dialog
+        :loading="!eventsAndSubjectsLoaded"
+        :event="dialogEvent || lastDialogState.event"
+        :user-is-member="userIsMember"
+        @close="closeEventDetailsDialog()"
+      />
+    </v-dialog>
+  </v-container>
+</template>
+
+<script>
+  import AppBar from '../components/AppBar.vue';
+  import EventList from '../components/board/EventList.vue';
+  import EventDetailsDialog from '../components/board/EventDetailsDialog.vue';
+  import EventCreateDialog from '../components/board/EventCreateDialog.vue';
+
+  export default {
+    name: 'BoardEvents',
+    components: {
+      AppBar,
+      EventList,
+      EventDetailsDialog,
+      EventCreateDialog,
+    },
+    data: () => ({
+      boardInfo: null,
+      boardInfoLoaded: false,
+      events: null,
+      subjects: null,
+      eventsAndSubjectsLoaded: false,
+      lastDialogState: {
+        event: null,
+        edit: false,
```

Comment:
lastDialogState.edit is unnecessary, as create is not used, please simplify to `lastDialogEvent`